src/data/faq.json

```diff
@@ -1052,35 +1054,42 @@
+                                    "The luca system transmits the contact data required for this purpose from visitors to the event or establishment in question to the health authorities as part of contact tracking, which must be health authority-compliant. However, the CWA does not provide for the transmission of contact data. Thus, QR codes were needed for this process, which were different from those required by the CWA."
                                 ]
                             },
                             {
                                 "title": "Can users of the luca app be alerted if CWA users test positive?",
                                 "anchor": "check_in_luca_warn",
                                 "active": false,
                                 "textblock": [
-                                    "No, not automatically. Even if users of both apps scan the same QR code, no data flows from one system to the other. Only when the responsible health office learns of the infection from a test laboratory, for example, can it personally inform the affected luca users about the possible risk contact. However, the principle of the CWA is based on the fact that users are warned pseudonymously via a 'red tile' (increased risk) in the event of a risk encounter. No interaction with a health department is necessary in this process."
+                                    "<b>Updated 15 July, 2022</b>",
+                                    "Note: Since spring 2022, the event registration (check-in functionality) of the luca app is no longer possible for the time being. Even when this functionality was still available in the luca app, luca app users were not automatically warned when CWA users tested positive.<hr>",
+                                    "Even if users of both apps scan the same QR code, no data flows from one system to the other. Only when the responsible health office was informed about the infection from a test laboratory, for example, it could have personally informed the affected luca users about the possible risk contact. However, the principle of the CWA is based on the fact that users are warned pseudonymously via a 'red tile' (increased risk) in the event of a risk encounter. No interaction with a health department is necessary in this process."
                                 ]
                             },
                             {
-                                "title": "Can all QR codes for check-in be read immediately by the Corona-Warn-App and the luca app?",
+                                "title": "Can all luca QR codes for check-in be read immediately by the Corona-Warn-App and the luca app?",

```

Comment:
What does "immediately" mean here? Can this word be removed?

Reply:
Removed in commit https://github.com/corona-warn-app/cwa-website/pull/2757/commits/0b254036e8d8d1fad6ce38c9d1614b4b9313aac2, thanks